cylc/flow/cfgspec/globalcfg.py

```diff
@@ -133,48 +133,50 @@
             Conf('ranking', VDR.V_STRING, desc='''
                 A multiline string containing Python expressions to filter
                 and/or rank hosts. For example:
-                cpu_percent() < 70
-                cpu_percent()
-                to filter by cpu_percent() < 70 then to rank by cpu_percent.
+
+                .. code-block:: none
+
+                   cpu_percent() < 70
+                   cpu_percent()
+
+                To filter by ``cpu_percent() < 70`` then to rank by
+                ``cpu_percent``.
             ''')
 
         with Conf('host self-identification', desc='''
             The suite host's identity must be determined locally by cylc and
             passed to running tasks (via ``$CYLC_SUITE_HOST``) so that task
             messages can target the right suite on the right host.
-
-            .. todo
-            Is it conceivable that different remote task hosts at the same site
-            might see the suite host differently? If so we would need to be
-            able to override the target in suite configurations.
         '''):
+            # TODO
+            # Is it conceivable that different remote task hosts at the same
+            # site might see the suite host differently? If so we would need to
+            # be able to override the target in suite configurations.
```

Comment:
We don't have the "TODO" directive in Sphinx, in RST files just use an RST comment, in Python files, use a regular comment.